cmd/podman/volumes/export.go

```diff
@@ -0,0 +1,77 @@
+package volumes
+
+import (
+	"context"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/inspect"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/utils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeExportDescription = `
+podman volume export
+
+Allow content of volume to be exported into external tar.`
+	exportCommand = &cobra.Command{
+		Use:               "export [options]",
+		Short:             "Export volumes",
+		Args:              cobra.ExactArgs(1),
+		Long:              volumeExportDescription,
+		RunE:              export,
+		ValidArgsFunction: common.AutocompleteVolumes,
+	}
+)
+
+var (
+	// Temporary struct to hold cli values.
+	cliExportOpts = struct {
+		Output string
+	}{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: exportCommand,
+		Parent:  volumeCmd,
+	})
+	flags := exportCommand.Flags()
+
+	outputFlagName := "output"
+	flags.StringVarP(&cliExportOpts.Output, outputFlagName, "o", "/dev/stdout", "Output=[path]")
```

Comment:
Please do not use `/dev/stdout` as path. This will fail on windows. Actually looking at this this will not work with podman-remote at all, right? I think not supporting remote would be fine but we should disable this command on the remote cleint and document this.

Reply:
I agree, no need for remote support.

Reply:
@Luap99 Yes we cant have this on remote-client. Do you still think we should not use `/dev/stdout` ?

Reply:
/dev/stdout is fine if we only support linux.

Look here how to disable to command on the remote client:
https://github.com/containers/podman/blob/7d8650cce82e67fb998b9f8a89c66a51b4c98285/cmd/podman/networks/reload.go#L18

Reply:
@Luap99 linux or *nix. I disabled it via `!registry.IsRemote()`.

Reply:
Please use the Annotation and not use `!registry.IsRemote()` This should be consistent with the other commands.

Reply:
@Luap99 Yes annotation seems cleaner switched to annotation, PTAL.